dropwizard-metrics-datadog/src/main/java/io/dropwizard/metrics/DatadogReporterFactory.java

```diff
@@ -34,5 +39,14 @@ public ScheduledReporter build(MetricRegistry registry) {
         .convertDurationsTo(getDurationUnit())
         .convertRatesTo(getRateUnit())
         .build();
-    }
+  }
+
+  @Override
+  public MetricFilter getFilter() {
```

Comment:
Seems like you could skip overriding `getFilter()` and the `useRegexFilters()` and just the following to the `build` function:
```java
MetricFilter filter = useRegexFilters ?
        new RegexMetricFilter(getIncludes(), getExcludes()) : getFilter();
```